src/RevCPU.cc

```diff
@@ -387,6 +378,22 @@ RevCPU::~RevCPU() {
   delete Opts;
 }
 
+std::vector<std::string> RevCPU::FindArgs( const SST::Params& params ) {
+  const char               delim[] = " \t\n";
+  std::vector<std::string> arglist;
+
+  // If the "args" param does not start with a left bracket, split it up at whitespace
+  // Otherwise interpet it as an array
+  std::string args     = params.find<std::string>( "args" );
+  auto        nonspace = args.find_first_not_of( delim );
+  if( nonspace == args.npos || args[nonspace] != '[' ) {
+    RevOpts::splitStr( args, delim, arglist );
+  } else {
+    params.find_array( "args", arglist );
+  }
+  return arglist;
+}
+
```

Comment:
It accomplishes what we talked about -- it first treats the `args` parameter as a string, and if it does not (after leading whitespace) start with a `[`, treats it the way that the old Rev and Tests handled it. If it starts with a `[`, the `args` parameter is treated as an array.

This is better than basing it on the number of arguments in the array, since sometimes you'll want only one argument, even if it has spaces in it.

The existing `argc` test uses the space-delimited format `"one two three"` and this change maintains backwards compatibility.